framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -538,6 +543,28 @@ private Range calculateLengthRangeAddition(Range leftLengths, Range rightLengths
         return leftLengths.plus(rightLengths).intersect(Range.INT_EVERYTHING);
     }
 
+    /**
+     * Checks whether or not the passed node is nullable. This superficial check assumes that every
+     * node is nullable unless it is a primitive, String literal, or compile-time constant.
```

Comment:
The comment and code will be easier to read if they both list the cases in the same order.

Reply:
@Maxi17 Could you please address this comment?

Reply:
I changed the order of checks in this method. Hopefully it's what was expected.